### ISSUE TITLE
Unable to install pylance

### ISSUE THREAD

**Issue by Sharkum** [microsoft/vscode#143662]
Issue Type: <b>Performance Issue</b>

TLDR :- Just unable to install pylance. 

1) I have a fresh installation of vscode and have no extensions. 
2) I install the python extension from the marketplace.
3) The extensions in the extension pack of python( jupyter extensions and pylance) all show that they are installing.
4) They get stuck at installing and never finish installing. 
5) If i close the window and check the extensions tab, all the extensions in the python extension pack are installed except for pylance. 
6) If i try to install pylance again, it shows that it is installing and never terminates.
7) I am able to install all the other extensions just fine( Intellicode, sql extensions etc.) 
I have asked the question on stackoverflow already, but haven't gotten any response yet. 

VS Code version: Code 1.64.2 (f80445acd5a3dadef24aa209168452a3d97cc326, 2022-02-09T22:02:28.252Z)
OS version: Windows_NT x64 10.0.19044
Restricted Mode: No

<details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|CPUs|AMD Ryzen 7 5800H with Radeon Graphics          (16 x 3194)|
|GPU Status|2d_canvas: enabled<br>gpu_compositing: enabled<br>multiple_raster_threads: enabled_on<br>oop_rasterization: enabled<br>opengl: enabled_on<br>rasterization: enabled<br>skia_renderer: enabled_on<br>video_decode: enabled<br>vulkan: disabled_off<br>webgl: enabled<br>webgl2: enabled|
|Load (avg)|undefined|
|Memory (System)|15.86GB (9.66GB free)|
|Process Argv|--crash-reporter-id dff5ef38-fb9a-4869-8803-72fd6fbc9324|
|Screen Reader|no|
|VM|0%|
</details><details>
<summary>Process Info</summary>

```
CPU %	Mem MB	   PID	Process
    0	    87	  7396	code main
    0	    25	  6224	   crashpad-handler
    0	    37	  8396	   utility-network-service
    0	    97	 11636	   shared-process
    0	    72	 15952	     fileWatcher
    0	    78	 17076	     ptyHost
    0	     6	  6272	       console-window-host (Windows internal process)
    0	    64	 16804	       C:\WINDOWS\System32\WindowsPowerShell\v1.0\powershell.exe
    0	   151	 13176	   window (q1.py - PDSA - Visual Studio Code)
    0	    81	 13652	   issue-reporter
    0	   131	 15364	   extensionHost
    0	    74	  2892	     "C:\Users\sharan\AppData\Local\Programs\Microsoft VS Code\Code.exe" --ms-enable-electron-run-as-node c:\Users\sharan\.vscode\extensions\mtxr.sqltools-0.23.0\dist\languageserver.js --node-ipc --clientProcessId=15364
    0	   114	 16992	   gpu-process
```

</details>
<details>
<summary>Workspace Info</summary>

```
|  Window (q1.py - PDSA - Visual Studio Code)
|    Folder (PDSA): 13 files
|      File types: py(8) txt(2)
|      Conf files:;
```

</details>
<details><summary>Extensions (7)</summary>

Extension|Author (truncated)|Version
---|---|---
vscode-sqlite|ale|0.14.0
python|ms-|2022.0.1814523869
jupyter|ms-|2022.1.1301854968
jupyter-keymap|ms-|1.0.0
jupyter-renderers|ms-|1.0.6
sqltools|mtx|0.23.0
vscodeintellicode|Vis|1.2.17

</details><details>
<summary>A/B Experiments</summary>

```
vsliv368cf:30146710
vsreu685:30147344
python383:30185418
vspor879:30202332
vspor708:30202333
vspor363:30204092
pythonvspyl392cf:30425750
pythontb:30283811
pythonptprofiler:30281270
vshan820:30294714
vstes263:30335439
pythondataviewer:30285071
vscod805cf:30301675
pythonvspyt200:30340761
binariesv615:30325510
bridge0708:30335490
bridge0723:30353136
vsaa593cf:30376535
vsc1dst:30438360
pythonvs932:30410667
wslgetstarted:30433507
vsclayoutctrc:30437038
vsrem710cf:30416617
vscop841:30438915
dsvsc008:30440022
pythonvspyt640:30442238
vsbas813:30436447
vscscmwlcmt:30438805
helix:30440343
vscaac:30438847

```

</details>

**Comment by sandy081** [comment]
Please try to install only Pylance extension and share with us following information:

- Enable log level to trace (F1 > Set Log Level...)
- Open Log (Shared Process) (F1 > Open View... > Log (Shared Process) and share the logs

**Comment by NChechulin** [comment]
No new info, but same here
